Consumer.py

```diff
@@ -0,0 +1,40 @@
+from kafka import KafkaConsumer
+from pymongo import MongoClient
+from flask_pymongo import PyMongo
+import json
+from flask_pymongo import PyMongo
+from Services.SessionAlertServices import postAlertData
```

Comment:
Had an issue trying to get this file file to run for two reasons 
1. The name of the service you're trying to import does not possess an s in the name it's SessionAlertService
2. You seemed to have removed the Flask import. We are getting this error now:
```
 File "Consumer.py", line 9, in <module>
    app= Flask(__name__)
NameError: name 'Flask' is not defined
```

Reply:
Added from flask import Flask in both scripts and tested functionality.
Fixed the typo in Services to Service.

Since I changed the topic name because this consumer is supposed to be receiving data from another microservice, currently, it would be from Producer.py in off-top-python, I renamed this topic to NewFocusAlert as suggested but not you can't really test the producer with the consumer because of the topic name mismatch.